.travis.yml

```diff
@@ -25,6 +25,7 @@ install:
 script:
   - nosetests --with-doctest --with-coverage
   - pep8 biom setup.py
+  - flake8 biom setup.py
```

Comment:
should probably run on `scripts/` as well assuming that's where the click command is going?